grafana-plugin/src/containers/AddResponders/AddResponders.styles.ts

```diff
@@ -0,0 +1,72 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getAddRespondersStyles = (theme: GrafanaTheme2) => {
+  return {
+    content: css`
+      width: 100%;
+      position: relative;
+    `,
+
+    respondersList: css`
+      list-style-type: none;
+      margin-bottom: 20px;
+      width: 100%;
+
+      & > li .hover-button {
```

Comment:
@joeyorlando I'm migrating more code to emotion and have come across this part, can't really figure out where's `hover-button` coming from as it's not being defined in any of the add responder styles. Do you happen to know where is that coming from? I know you've been working on this stuff

The only `hover-button` I see is part of the integration display within the routes, but that shouldn't be related to add responders in any way